backend/attachRendererFiber.js

```diff
@@ -42,35 +44,26 @@ function attachRendererFiber(hook: Hook, rid: string, renderer: ReactRenderer):
   }
 
   function hasDataChanged(prevFiber, nextFiber) {
-    if (prevFiber.tag === ClassComponent) {
-      // Skip if the class performed no work (shouldComponentUpdate bailout).
-      // eslint-disable-next-line no-bitwise
-      if ((nextFiber.effectTag & PerformedWork) !== PerformedWork) {
-        return false;
-      }
-
-      // Only classes have context.
-      if (prevFiber.stateNode.context !== nextFiber.stateNode.context) {
-        return true;
-      }
-      // Force updating won't update state or props.
-      if (nextFiber.updateQueue != null && nextFiber.updateQueue.hasForceUpdate) {
```

Comment:
Don't need this because "performed work" includes forced updates (as long as we ran render, we performed work).